samples/application_development/event_manager/prj.conf

```diff
@@ -0,0 +1,19 @@
+#
+# Copyright (c) 2019 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+# Enabling assert
+CONFIG_ASSERT=y
+
+# Logger configuration
+CONFIG_LOG=y
+CONFIG_LOG_DEFAULT_LEVEL=2
+CONFIG_LOG_BACKEND_UART=y
+CONFIG_LOG_STRDUP_BUF_COUNT=10
+
+# Configuration required by Event Manager
+CONFIG_EVENT_MANAGER=y
+CONFIG_LINKER_ORPHAN_SECTION_PLACE=y
+CONFIG_HEAP_MEM_POOL_SIZE=2048
+CONFIG_REBOOT=y
```

Comment:
why do we need this?

Reply:
We run sys_reboot if there is no enough memory to allocate event.

Reply:
Why not wait for the memory to be released? Is it due to support for single thread?

Reply:
We cannot drop events as it would lead to unexpected situations. If such problems happens it is safer to reboot.